packages/drivechain_client/lib/pages/overview_page.dart

```diff
@@ -85,16 +85,17 @@ class BalancesView extends StatelessWidget {
                       children: [
                         SailText.primary13('Available: '),
                         RichText(
-                            text: TextSpan(
-                          text: Money.fromNumWithCurrency(
-                            snapshot.data!.confirmedSatoshi.toDouble(),
-                            satoshi,
-                          ).toString(),
-                          style: SailStyleValues.thirteen.copyWith(
-                            color: theme.colors.text,
-                            fontWeight: FontWeight.bold,
+                          text: TextSpan(
+                            text: Money.fromNumWithCurrency(
+                              snapshot.data!.confirmedSatoshi.toDouble(),
+                              satoshi,
+                            ).toString(),
+                            style: SailStyleValues.thirteen.copyWith(
+                              color: theme.colors.text,
+                              fontWeight: FontWeight.bold,
+                            ),
```

Comment:
I would like to never drag `SailStyleValues.thirteen.copyWith` into the view. That is a highway to 100 different text styles. I'd prefer to keep it to a minimum.

First check if there's something you can use from the existing styles. If there's really nothing to use, add a new `SailText.somethingsomething`


```suggestion


                        SailText.primary13(
                          Money.fromNumWithCurrency(
                            snapshot.data!.confirmedSatoshi.toDouble(),
                            satoshi,
                          ).toString(),
                          bold: true,
                        ),
```